src/reference/antora/modules/ROOT/pages/mail.adoc

```diff
@@ -272,12 +395,107 @@ mail.smtp.port=587
 
 NOTE: As with any outbound Channel Adapter, if the referenced channel is a `PollableChannel`, you should provide a `<poller>` element (see xref:endpoint.adoc#endpoint-namespace[Endpoint Namespace Support]).
 
-When you use the namespace support, you can also use a `header-enricher` message transformer.
+You can also use a `header-enricher` message transformer.
```

Comment:
What is your opinion about having technical doc impersonated?
I fight for that for years, but looks like everyone continues using `you`, `we`, `their` etc...